e2e/config/azure.go

```diff
@@ -448,7 +448,14 @@ func (a *AzureClient) ensureReplication(ctx context.Context, t *testing.T, image
 		return nil
 	}
 	t.Logf("##vso[task.logissue type=warning;]Replicating to region %s: image version %s", Config.Location, *version.ID)
-	return a.replicateImageVersionToCurrentRegion(ctx, image, version)
+
+	start := time.Now() // Record the start time
+	err := a.replicateImageVersionToCurrentRegion(ctx, image, version)
+	elapsed := time.Since(start) // Calculate the elapsed time
+
+	fmt.Printf("##vso[task.logissue type=warning;]Replication took: %s (%s)\n", elapsed, *version.ID)
```

Comment:
why is this a warning ?

Reply:
I tried to find docs, and the only two options I could see were "warning" and "error". Happy to change if you can find one for info!

Reply:
but what's the intention here ? warning print in orange in the log, error prints in red. 

why do we want this information to pop out ?

Reply:
Have changed to only log as a warning if the duration is over a limit.

It turns out "information" didn't work - only warning and error are valid types.